src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -260,9 +259,9 @@ end
 --     of those cards which will be spawned
 function maybeAddOnTheMend(slots)
   if slots["09006"] ~= nil then
-    local playArea = getObjectFromGUID(PLAY_AREA_GUID)
-    if playArea ~= nil then
-      slots["09006"] = playArea.call("getInvestigatorCount")
+    local investigatorCount = playAreaApi.getInvestigatorCount()
+    if investigatorCount ~= nil then
```

Comment:
If we are checking this anyway, there should probably be a message if it's not found.

Reply:
Done, and made it spawn 4 as a backup.  Though if this fails the game's pretty much irredeemably broken so On the Mend may be the only thing they get out of it...